apps/auction2/client.js

```diff
@@ -70,12 +74,17 @@ const params = new Params(defaultParams, {
 })
 const filter = new Filter({ params: params })
 
-// Main Artworks view
+// REDUX
+const store = createStore(auctions)
```

Comment:
The artworks load client-side, so there isn't anything to hydrate the store with at this point. They'll be added in the `filter.artworks.on('reset')` event.